File: bench/primes1.lua
local upto = 20000
local ret = 0
local at = 2
while at < upto do
    local is_prime = true
    test = 2
    while test < at do
        if at % test < 1 then
            is_prime = false
        end
        test = test + 1
    end
    if is_prime then
        ret = ret + 1
    end
    at = at + 1
end

print(ret)

File: bench/primes1.js
var upto = 50000;
var ret = 0;
var at = 2;
while (at < upto) {
    var is_prime = true;
    var test = 2;
    while (test < at) {
        if (at % test < 1) {
            is_prime = false;
        }
        test = test + 1;
    }
    if (is_prime) {
        ret = ret + 1;
    }
    at = at + 1;
}

console.log(ret);